test/ruby/test_regexp.rb

```diff
@@ -1813,4 +1813,16 @@ def test_linear_performance
       re =~ s
     end
   end
-end
+
+  def test_timeout_setter
+    re = /(a)\1{100000}/.dup
+    re.timeout = 0.1
+    assert_equal(0.1, re.timeout)
+    assert_raise(Regexp::TimeoutError) {re.match? "a"* 100000}
+  end
+
+  def test_timeout_setter_frozen
+    re = /(a)\1{100000}/
+    assert_raise(FrozenError) {re.timeout = 0.1}
+  end
+end
```

Comment:
Why removed a newline?

Reply:
whoops! this happened when I was trying to fix an unrelated whitespace issue.